### DISCUSSION TITLE
Output: 530 Authentication required

### DISCUSSION THREAD

**Discussion by jarqvi** [postalserver/postal#3069] [Help with using Postal]
## Describe the bug
Hey everyone.
I registered a domain in Postal with two addresses, like:
```
domain: example.com
address-1: info@example.com
address-2: contact@example.com
```
When i want send an email from address-1(info@example.com) to address-2(contact@example.com) i get this error:
```
Temporary SMTP delivery error when sending to 88.X.X.X:25 (XXX.email)
Output: 530 Authentication required
```

Whereas if I have registered another domain in Postal, for example:
```
domain: domain.com
address-1: info@domain.com
address-2: contact@domain.com
```

I can send an email from info@example.com to contact@domain.com.
Does anyone have a solution?

**Comment by willpower232** [comment]
I'm confused about your setup, are you trying to send an email from Postal to the same Postal?

**Comment by jarqvi** [comment]
I have two addresses from the same domain on a postal.
I get this error when I send email from the first address to the second address of this domain.

```
Temporary SMTP delivery error when sending to 88.X.X.X:25 (XXX.email)
Output: 530 Authentication required
```
